### PR TITLE
Monorepo : added rest-client & rest-express to the monorepo

### PR DESCRIPTION
**Pull request by Aetherall**

### CODE REVIEW

**Comment by pradel** [comment]
Looks like codecov didn't work for that pr 🤔 https://codecov.io/gh/accounts-js/accounts/pulls

**Comment by Aetherall** [comment]
Strange, It does see the branch with the commits
https://codecov.io/gh/accounts-js/accounts/branch/monorepo%2Frest

**Comment by pradel** [comment]
Does not really matter the coverage is 92+% on that branch

File: packages/rest-express/package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "keywords": [
+    "users",
+    "accounts",
+    "rest",
+    "express"
+  ],
+  "author": "Tim Mikeladze",
+  "license": "MIT",
+  "devDependencies": {
+    "@accounts/common": "0.1.0-beta.3",
```
Comment (pradel) [inline packages/rest-express/package.json]: the types definitions are missing there

Comment (pradel) [reply packages/rest-express/package.json]: originaly:
```
"devDependencies": {
    "@accounts/common": "0.1.0-beta.3",
    "@accounts/server": "0.1.0-beta.3",
    "@types/express": "4.11.1",
    "@types/lodash": "4.14.104",
    "@types/request-ip": "0.0.33"
  },
```

Comment (Aetherall) [reply packages/rest-express/package.json]: It's fixed :)

**Comment by pradel** [comment]
Thanks @Aetherall

**Pull request merged**